testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -46,6 +47,11 @@ def pca(u):
     u.transfer_to_memory()
     return PCA(u, select=SELECTION).run()
 
+@pytest.fixture(scope='module')
+def pca_aligned(u):
+    u.transfer_to_memory()
```

Comment:
Is this necessary? I think PCA fits per frame when aligning.

Reply:
I think it is for speed purposes, however i have removed it as suggested as it strengthens the test.